.github/dependabot.yml

```diff
@@ -9,6 +9,12 @@ updates:
     time: "17:00"
     timezone: America/Los_Angeles
   open-pull-requests-limit: 10
+  groups:
+    wp-cli-packages:
+      patterns:
+        - "wp-cli/*"
+  ignore:
+    - dependency-name: phpdocumentor/reflection
```

Comment:
Could you add a comment to this explaining why it is ignored?

Reply:
Done in https://github.com/ampproject/amp-wp/pull/7670/commits/34bed2f08699c6fbf536243f61147ccf0770e525.